### DISCUSSION TITLE
Geospatial example

### DISCUSSION THREAD

**Discussion by 870s3d** [blue-yonder/tsfresh#1044] [Q&A]
Are there any examples for using `tsfresh` on geospatial data (i.e.: lat/lon or geohash)?

**Comment by nils-braun** [comment]
Hi @870s3d - are you asking because you are not sure about the technical details (how to use tsfresh) or because you are not sure if it gives reasonable results? I can not comment on the latter - but for the first question: there should not be any difference in usage. Your time series in this case should be your metric vs one spatial dimension (equidistant binning is preferred), e.g. vs. lat. You could have different lon values as different time series (or transposed, or both).
